tests/models/sam/test_modeling_sam.py

```diff
@@ -733,3 +733,20 @@ def test_inference_mask_generation_three_boxes_point_batch(self):
         iou_scores = outputs.iou_scores.cpu()
         self.assertTrue(iou_scores.shape == (1, 3, 3))
         torch.testing.assert_allclose(iou_scores, EXPECTED_IOU, atol=1e-4, rtol=1e-4)
+
+    def test_post_process_masks(self):
```

Comment:
This isn't a model test - it should be with the processing tests

Reply:
Yes and no, it need the outputs from a model

Reply:
I could! Just wanted to make sure that the post processing performs what we want it to perform.
Will add dummy masks.

Reply:
Also it was good to test the `reshaped_input_sizes` that were in tensor format (from being outputed by the model).
So I still wanna make sure that the post process can run from the outputs of the model.